jp/lessons/libraries/poolboy.md

```diff
@@ -0,0 +1,161 @@
+---
+version: 1.1.1
+title: Poolboy
+redirect_from:
+  - /lessons/libraries/poolboy/
```

Comment:
Please remove this redirect

Reply:
I will take this pr, so please give me more time ;)